public/request/leaderboard/update.php

```diff
@@ -18,13 +18,26 @@
 
 getCookie($user, $cookie);
 
-//	Somewhat elevated privileges to modify an LB:
+//  Somewhat elevated privileges to modify an LB:
 if (validateFromCookie($user, $points, $permissions, \RA\Permissions::Developer)
     && $source == $user) {
+    $prevData = GetLeaderboardData($lbID, $user, 1, 0, false);
+    $prevUpdated = $prevData["Updated"];
+
     if (submitLBData($user, $lbID, $lbMem, $lbTitle, $lbDescription, $lbFormat, $lbLowerIsBetter, $lbDisplayOrder)) {
         echo "OK";
-        $commentText = 'made updates to this leaderboard';
-        addArticleComment("Server", \RA\ArticleType::Leaderboard, $lbID, "\"$user\" $commentText.", $user);
+
+        $updatedData = GetLeaderboardData($lbID, $user, 2, 0, false);
+        $entries = $updatedData['Entries'];
+        $updated = $updatedData["Updated"];
+        $dateDiffMins = ($updated - $prevUpdated) / 60;
+
+        if (count($updatedData['Entries']) > 0) {
```

Comment:
Could reuse the assigned `$entries` from above or remove the unused variable.
Using `!empty($entries)` instead of `count($entries)` will not actually count and thus is faster. Won't have much of an impact in this case though.